internal/host/hostcommands/instruction_manager.go

```diff
@@ -117,7 +118,6 @@ func NewInstructionManager(log logrus.FieldLogger, db *gorm.DB, hwValidator hard
 			models.HostStatusDisabledUnbound:     {[]CommandGetter{}, defaultBackedOffInstructionInSec, models.StepsPostStepActionContinue},
 			models.HostStatusInsufficientUnbound: {[]CommandGetter{inventoryCmd}, defaultNextInstructionInSec, models.StepsPostStepActionContinue},
 			models.HostStatusKnownUnbound:        {[]CommandGetter{inventoryCmd}, defaultNextInstructionInSec, models.StepsPostStepActionContinue},
-			models.HostStatusBinding:             {[]CommandGetter{noopCmd}, 0, models.StepsPostStepActionExit},
```

Comment:
Why did you remove this part? 
How will day2 works with late binding? 
We have a host registered to InfraEnv now we bind it to a day2 cluster what will be the flow?

Reply:
it is the same flow as day1. We need to add a Binding state to addHostsClusterToSteps instruction manager set (MGMT-7757)

Reply:
in any case, there can not be state Binding for unbound host, since once host is moved to stateBinding, it becomes Bound host (no matter if day1 or day2